pkgs/misc/screensavers/i3lock-pixeled/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, pkgs, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "i3lock-pixeled-${version}";
+  version = "1.1.0";
+
+  src = fetchurl {
+    url = "https://github.com/Ma27/i3lock-pixeled/archive/${version}.tar.gz";
+    sha256 = "046qbx4qvcc66h53h4mm9pyjj9gjc6dzy38a0f0jc5a84xbivh7k";
+  };
+
+  buildInputs = with pkgs; [ i3lock imagemagick scrot playerctl ];
```

Comment:
I think that `buildInputs` should be renamed `propagatedBuildInputs` since all those programs are required at run-time and not present otherwise.